testutil/blobfactory/payforblob_factory.go

```diff
@@ -377,6 +376,50 @@ func MultiBlobTx(
 	return cTx
 }
 
+// IndexWrapperWithInvalidNamespace returns an index wrapped PFB tx with an
+// invalid namespace and a blob associated with that index wrapped PFB tx.
+func IndexWrapperWithInvalidNamespace(
+	t *testing.T,
+	enc sdk.TxEncoder,
+	signer *blobtypes.KeyringSigner,
+	sequence uint64,
+	accountNum uint64,
+	index uint32,
+) (coretypes.Tx, tmproto.Blob) {
+	addr, err := signer.GetSignerInfo().GetAddress()
+	require.NoError(t, err)
+
+	coin := sdk.Coin{
+		Denom:  bondDenom,
+		Amount: sdk.NewInt(10),
+	}
+	opts := []blobtypes.TxBuilderOption{
+		blobtypes.SetFeeAmount(sdk.NewCoins(coin)),
+		blobtypes.SetGasLimit(10000000),
+	}
+
+	blob := ManyRandBlobs(t, 100)[0]
```

Comment:
?
```suggestion
	blob := ManyRandBlobs(t, 1)[0]
```

Reply:
`100` is the blob size

https://github.com/rootulp/celestia-app/blob/93f74344b7394173d264151628b5fe076c57c9a0/testutil/blobfactory/payforblob_factory.go#L288

https://github.com/rootulp/celestia-app/blob/93f74344b7394173d264151628b5fe076c57c9a0/testutil/blobfactory/payforblob_factory.go#L300

Reply:
lol :face_in_clouds: